packages/repco-webcomponent/.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
can this file be removed? we already have a eslint setup. let's try to not add different setups to different packages.